WalletWasabi/Tor/Socks5/TorTcpConnection.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.IO;
+using System.Net.Security;
+using System.Net.Sockets;
+
+namespace WalletWasabi.Tor.Socks5
+{
+	/// <summary>
+	/// Wraps a TCP connection to Tor SOCKS5 endpoint.
+	/// </summary>
+	public class TorTcpConnection : IDisposable
```

Comment:
explanation: This wraps a single connection to Tor SOCKS5 endpoint and it will allow us to add tests for the coming Tor pool where you will more than one `TorTcpConnection` at a time.